ui/app/js/controllers/scheduled_release_update_controller.js

```diff
@@ -0,0 +1,18 @@
+angular.module('app').controller('ScheduledReleaseUpdateDataCtrl',
```

Comment:
Just a couple of minor nits on this:
1) Please improve the name - the current one makes it sound like it's for updating scheduled releases. "ScheduledReleaseDiffCtrl" and "scheduled_release_diff_controller.js" would be better.
2) Please improve the formatting to make the indentation match the rest of the files.